rotkehlchen/tests/unit/decoders/test_metamask.py

```diff
@@ -0,0 +1,727 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.arbitrum_one.modules.metamask.constants import (
+    METAMASK_ROUTER as METAMASK_ROUTER_ARB,
+)
+from rotkehlchen.chain.ethereum.modules.metamask.constants import (
+    METAMASK_ROUTER as METAMASK_ROUTER_ETH,
+)
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.decoding.metamask.constants import CPT_METAMASK
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.chain.optimism.modules.metamask.constants import (
+    METAMASK_ROUTER as METAMASK_ROUTER_OPT,
+)
+from rotkehlchen.chain.polygon_pos.modules.metamask.constants import (
+    METAMASK_ROUTER as METAMASK_ROUTER_MATIC,
+)
+from rotkehlchen.constants.assets import A_AAVE, A_ETH, A_POLYGON_POS_MATIC, A_USDC, A_USDT
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.constants import A_OPTIMISM_USDT
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+A_LUX = Asset('eip155:1/erc20:0x88dafebb769311d7fbbeb9a21431fa026d4100d0')
+A_OTACON = Asset('eip155:1/erc20:0x0F17eeCcc84739b9450C88dE0429020e2DEC05eb')
+A_INU = Asset('eip155:1/erc20:0xc76d53f988820fe70e01eccb0248b312c2f1c7ca')
+A_MOG = Asset('eip155:1/erc20:0xaaeE1A9723aaDB7afA2810263653A34bA2C21C7a')
+A_INJ = Asset('eip155:1/erc20:0xe28b3B32B6c345A34Ff64674606124Dd5Aceca30')
+A_ARBITRUM_USDC = Asset('eip155:42161/erc20:0xaf88d065e77c8cC2239327C5EDb3A432268e5831')
+A_OPTIMISM_USDC = Asset('eip155:10/erc20:0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85')
+A_POLYGON_USDC = Asset('eip155:137/erc20:0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174')
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x7e6Ee5d52825B3A3d9C500E7B8b0a2BAa1c91545']])
+def test_metamask_swap_token_to_eth(database, ethereum_inquirer, ethereum_accounts):
+    """Data taken from
+    https://etherscan.io/tx/0x9b675024d8648c3b590eff411fcf75a1199d10d1a3fe2ddbe50e166ce8b87cc9
+    """
```

Comment:
In this and the other decoder tests please don't write this. It's redundant info

```suggestion
```